src/app/resources/resources.component.ts

```diff
@@ -40,4 +48,28 @@ export class ResourcesComponent implements OnInit {
         this.resources = data.rows;
       }, error => (this.message = 'Error'));
   }
+
+  getNationResources() {
+    if (this.route.snapshot.paramMap.get('nationname') !== null) {
+      this.couchService.get('nations/_all_docs?include_docs=true')
+      .then((data) => {
+         data.rows.map(function(nt){
+          if (nt.doc.name === this.route.snapshot.paramMap.get('nationname')) {
+            this.nationname = this.route.snapshot.paramMap.get('nationname');
+            const nationUrl = nt.doc.nationurl;
+            if (nationUrl) {
+              this.jsonp.request('http://' + nationUrl + '/resources/_all_docs?include_docs=true&callback=JSONP_CALLBACK&limit=5')
```

Comment:
for testing purpose i have limited the docs

Reply:
Also we need pagination in view page.